sqlglot/tokens.py

```diff
@@ -578,6 +580,7 @@ class Tokenizer(metaclass=_Tokenizer):
         "UNIQUE": TokenType.UNIQUE,
         "STRUCT": TokenType.STRUCT,
         "VARIANT": TokenType.VARIANT,
+        "SAMPLE": TokenType.TABLE_SAMPLE,
```

Comment:
i think this is problematic, what happens if you do

select sample from foo. will that translate to select table_sample?

Reply:
SAMPLE is a reserved word in Snowflake. I can make this a Snowflake dialect specific token.

Reply:
sounds good